content/docs/ref/send-comment.md

```diff
@@ -28,6 +28,17 @@ Any [generic option](/doc/ref) in addition to:
 
 ## FAQs and Known Issues
 
+### GitHub
+
+- **`commit_id` has been locked**.
+
+  This
+  [error](https://github.community/t/comment-api-does-not-describe-commit-id-has-been-locked/159853/2)
+  is caused by using the default GitHub token with `cml send-comment --update`.
+  Use a
+  [personal access token (PAT)](/doc/self-hosted-runners?tab=GitHub#personal-access-token)
```

Comment:
```suggestion
  [personal access token](/doc/self-hosted-runners?tab=GitHub#personal-access-token)
```

Subjective nitpick

Reply:
used in a few other places, probably best as-is for consistency :)